src/partials/resource-limits.md

```diff
@@ -0,0 +1,62 @@
+## Check resource usage {% #check-resource-usage %}
+
+You can check your resource usage by navigating to the overview of your organization. 
+Under the **Usage** tab, you can check current billing cycle's usage.
+
+
+## Reaching resource limits {% #reaching-resource-limits %}
+The following limit consequences will persist until the current billing period ends.
+{% table %}
+* Component
+* Consequence
+---
+* **Bandwidth** {% rowspan=3 %}
+* Project creation is disabled.
```

Comment:
when bandwidth limit is reached, all API access will be disabled. But for pro/scale teams, nothing will be disabled rather additional charges will apply, until their budget cap is reached only then it will be disabled.